jsk_robot_common/jsk_robot_startup/scripts/smach_to_mail.py

```diff
@@ -167,6 +167,8 @@ def _send_mail(self, subject, state_list):
 
         if subject:
             email_msg.subject = subject
+        elif rospy.has_param("/email_topic/mail_title"):
```

Comment:
Is `/email_topic/mail_tilte` used for  app_manager? if so, I think other rosparam uses private namespace such as
```
        if rospy.has_param("~sender_address"):
```
so I think this line also follows this convention. Please also update https://github.com/jsk-ros-pkg/jsk_robot/tree/master/doc/jsk_robot_common/jsk_robot_startup (I also forget to add ~sender_address and others...)

Reply:
Yes, `/email_topic/mail_title` is set by `app_manager`.
So, you mean we should unify namespace rules such as `/email_topic/sender_address` instead of `~sender_address`?